arm:
  type: ur10
sensors:
  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [-0.748136, 0.200000, 0.982218]
      rpy: [0, 0.725374, 0]
  logical_camera_2:
    type: logical_camera
    pose:
      xyz: [-0.748136, 1.0, 0.982218]
      rpy: [0, 0.725374, 0]

